packages/crypto/src/KeyType.ts

```diff
@@ -0,0 +1,9 @@
+export const ECC_COMPACT_KEY_TYPE = 0
```

Comment:
Could we share these files between crypto and crypto-react-native?

Reply:
I'd like to but there are some subtle differences because react-native uses a different sodium library. I'd like to maybe refactor the common stuff out into a separate package and then have the `crypto` packages just export a consistent api for the sodium packages. Just haven't gotten around to doing that yet